docs/bugs_and_glitches.md

```diff
@@ -246,7 +246,34 @@ DefenseDownHit:
 
 ([Video](https://www.youtube.com/watch?v=uRYyzKRatFk))
 
-*To do:* Identify specific code causing this bug and fix it.
+This is a bug with `BattleCommand_Counter` in [engine/battle/move_effects/counter.asm](/engine/battle/move_effects/counter.asm) and `BattleCommand_MirrorCoat` in [engine/battle/move_effects/mirror_coat.asm](/engine/battle/move_effects/mirror_coat.asm):
+
+```asm
```

Comment:
Oh, and this should be in diff format.

Reply:
Changed it to the diff format. But I think it should be noted that none of the other bugfixes use the diff format.